### PR TITLE
Update of options in utils and ddg2excel, addition of csv output

### LINKED ISSUE #87: The output text format isn't very user friendly
We should consider switching to something like a single, large table similar to the output of `ddg2excel`. Advantages:

* easier to understand from the point of view of the user
* easier to parse
* compatible with Pandas and hence easier to slice
* more compact

### PR DESCRIPTION
**Pull request by DegnKristine**
fixes #87, creates the opportunity to output a csv file in the ddg2excel script. This includes an updated description of the  -b LABELS option described in utils.py.

### CODE REVIEW

File: bin/ddg2excel [hunk 1]
```diff
@@ -26,12 +26,14 @@ from mutatex.utils import *
...
 LOGFMT = "%(levelname)s: %(message)s"
 
-description = 'ddg2excel: write the average values from a MutateX DDG scan as Excel tables'
-epilog = """ddg2excel writes the average values from a MutateX DDG scan as an Excel table
-file (.xlsx). Option -T swaps rows and columns of the output file. By default,
-residue type, number and chain identifier are written in separate columns/rows
-to make the data easily explorable, however one column/row is used when the
-label file is provided (option -b)"""
+description = 'ddg2tabel: writes the average values from a MutateX DDG scan as tables either in excel or in a csv file format'
+epilog = """ddg2tabel writes the average values from a MutateX DDG scan as a
```
Comment (mattu90) [inline bin/ddg2excel:30]: If I remember correctly, Matteo and Elena didn't want to change the tool name righT

File: bin/ddg2excel [hunk 2]
```diff
@@ -26,12 +26,14 @@ from mutatex.utils import *
...
-description = 'ddg2excel: write the average values from a MutateX DDG scan as Excel tables'
-epilog = """ddg2excel writes the average values from a MutateX DDG scan as an Excel table
-file (.xlsx). Option -T swaps rows and columns of the output file. By default,
-residue type, number and chain identifier are written in separate columns/rows
-to make the data easily explorable, however one column/row is used when the
-label file is provided (option -b)"""
+description = 'ddg2tabel: writes the average values from a MutateX DDG scan as tables either in excel or in a csv file format'
+epilog = """ddg2tabel writes the average values from a MutateX DDG scan as a 
+excel compatible file. Optional Option -F, --format chooses the output file 
+format, csv or xlsx, by default xlsx is choosen. Option -T swaps rows and
```
Comment (mattu90) [inline bin/ddg2excel:32]: just a typo "chosen" not "choosen"

File: mutatex/utils.py [hunk 3]
```diff
@@ -67,7 +67,7 @@ def init_arguments(arguments, parser):
         elif arg == 'multimers':
             parser.add_argument("-M","--multimers", dest="multimers", default=True, action='store_false', help="Do not use multimers (default: yes)")
         elif arg == 'labels':
-            parser.add_argument("-b","--label-list", dest="labels", help="Residue label list")
+            parser.add_argument("-b","--label-list", dest="labels", help="Residue label list generated by the pdb2labels")
```
Comment (mattu90) [inline mutatex/utils.py:70]: again, maybe it's just me but I would omit the "the" before pdb2labels

File: bin/ddg2excel (hunk 1 above)
Comment (DegnKristine) [reply bin/ddg2excel:30]: Corrected. Thank you!

File: bin/ddg2excel (hunk 2 above)
Comment (DegnKristine) [reply bin/ddg2excel:32]: Corrected. Thank you!

**Review by mattu90 — APPROVED** [review]

**Pull request merged**